script.js

```diff
@@ -116,29 +95,22 @@ function addLetter(letter, wordNum) {
 
 function handleGuess() {
   //the rest of the boxes should be hidden. This unhides the next box, and determines which letters were right / close / wrong
-  if (currentWord == 1) {
-    currentWord++;
-    checkWord(word1, 'row1');
-    unhideElement('row2');
-  } else if (currentWord == 2) {
-    currentWord++;
-    checkWord(word2, 'row2');
-    document.getElementById('row3').hidden=false;
-  } else if (currentWord == 3) {
-    currentWord++;
-    checkWord(word3, 'row3');
-    document.getElementById('row4').hidden=false;
-  } else if (currentWord == 4) {
-    currentWord++;
-    checkWord(word4, 'row4')
-    document.getElementById('row5').hidden=false;
-  } else if (currentWord == 5) {
-    currentWord++;
-    checkWord(word5, 'row5');
-    gameOver(false);
+  if(currentWord == 5){
```

Comment:
I'm confused why these two lines are here.

Reply:
Yes, this was left over from when I was testing something. Just was sloppy and forgot to delete

Reply:
Actually I do need an if statement for word5 to start the loss sequence. Just needed to have game over instead of check word